src/components/v2/Slider/index.tsx

```diff
@@ -33,19 +41,53 @@ export const Slider = ({
 
   const styles = useStyles({ over });
 
+  const renderMark = (props?: NonNullable<SliderTypeMap['props']['componentsProps']>['mark']) => {
+    if (markTooltip) {
+      return (
+        <span {...props} css={[styles.mark, styles.hasTooltip]}>
+          <Tooltip title={markTooltip}>
+            <span css={styles.tooltipHelper}>.</span>
+          </Tooltip>
+        </span>
+      );
+    }
+
+    return <span {...props} css={styles.mark} />;
+  };
+
+  const renderTrack = (props?: NonNullable<SliderTypeMap['props']['componentsProps']>['track']) => {
+    if (trackTooltip) {
+      return (
+        <div style={props?.style} css={[styles.trackWrapper, styles.hasTooltip]}>
+          <Tooltip title={trackTooltip}>
+            <Box {...props} style={{ width: '100%' }} />
```

Comment:
Let's not have inline styles

Reply:
it has to be inlined in order to reassign inline styles. left a comment here.
![image](https://user-images.githubusercontent.com/14794071/162692512-8f11e0c6-bbd5-4394-9098-d338d916f743.png)

Reply:
passed undefined and left a comment